filename: 124io_utils_test.py, smelly_lines: 25 (Too general exception)
filename: 171babi_rnn.py, smelly_lines: 162 (Too general exception)
filename: 179babi_memnn.py, smelly_lines: 101 (Too general exception)
filename: 41test_utils.py, smelly_lines: 23 (Too general exception)
filename: 49vis_utils.py, smelly_lines: 188 (Empty exception detected)
filename: 57cntk_backend.py, smelly_lines: 2612 (Too general exception)
filename: 63resnext.py, smelly_lines: 7 (Too general exception)
filename: 66resnet.py, smelly_lines: 7 (Too general exception)
filename: 72resnet_v2.py, smelly_lines: 7 (Too general exception)
filename: 94test_model_saving.py, smelly_lines: 26 (Too general exception)
filename: 98test_callbacks.py, smelly_lines: 28 (Too general exception)
